test/gtest/common/test_helpers.cc

```diff
@@ -76,17 +76,39 @@ bool is_inet_addr(const struct sockaddr* ifa_addr) {
 
 bool is_ib_netdev(const char *ifa_name) {
```

Comment:
can you please update the documentation of this function in the .h file now that it also checks guid?